cdap-docs/application-templates/source/etl/index.rst

```diff
@@ -30,16 +30,16 @@ an ETL Pipeline:
 ETL Templates, Adapters and Plugins 
 -----------------------------------
 
-An *Application Template* is a blueprint that is used to create an Adapter, an instantiation of
+An *application template* is a blueprint that is used to create an adapter, an instantiation of
 a template in CDAP.
 
-CDAP provides two Application Templates |---| the ETL Templates **ETLBatch** and
-**ETLRealtime** |---| which are used to create Adapters that perform ETL in either batch
-and realtime. The  *ETLBatch* and *ETLRealtime* templates consist of a variety of sources,
+CDAP provides two application templates |---| the ETL templates **ETLBatch** and
+**ETLRealtime** |---| which are used to create adapters that perform ETL in either batch
+and real-time. The  *ETLBatch* and *ETLRealtime* templates consist of a variety of sources,
```

Comment:
should this be "or" instead of "and"? Also, here I would use real time (without hyphen).